--- a/drivers/net/phy/broadcom.c
+++ b/drivers/net/phy/broadcom.c
@@ -58,12 +58,12 @@ static void bcm54213pe_config_init(struct phy_device *phydev)
 
 	/* Enable ACT+LINK indication on ACTIVITY trigger */
 	val = bcm_phy_read_shadow(phydev, BCM54XX_SHD_LEDCTL);
-	val |= BCM54XX_SHD_LEDCTL_ACTLINK_EN;
+	val &= ~BCM54XX_SHD_LEDCTL_ACTLINK_EN;
 	bcm_phy_write_shadow(phydev, BCM54XX_SHD_LEDCTL, val);
 
 	/* Set ACTIVITY on LED "1" output, LINKSPD[1] on LED "3" output */
-	val = BCM5482_SHD_LEDS1_LED1(BCM_LED_SRC_ACTIVITYLED) |
-		BCM5482_SHD_LEDS1_LED3(BCM_LED_SRC_LINKSPD1);
+	val = BCM5482_SHD_LEDS1_LED1(BCM_LED_SRC_OFF) |
+		BCM5482_SHD_LEDS1_LED3(BCM_LED_SRC_OFF);
 	bcm_phy_write_shadow(phydev, BCM5482_SHD_LEDS1, val);
 }
 